roles/openshift-applier/tasks/process-content.yml

```diff
@@ -4,10 +4,3 @@
   include_tasks: process-one-entry.yml
   with_items:
   - "{{ entry.content | default([]) }}"
-
```

Comment:
Can we please deprecate rather than cold-turkey remove it. I.e.: Keep this code, but add a *big fat warning* message about this going away in a near future release.

Reply:
@etsauer I'd do something with the `block` feature for the deprecated message, maybe something like the following (just update the message a bit):

```
- name: "Apply OpenShift Cluster Content - based on directory"
  block:
    - fail:
        msg: "This is deprecated ..."
      ignore_errors: True
    - command: >
        oc apply -f {{ tmp_inv_dir }}{{ entry.content_dir }}
  when:
  - entry.content_dir is defined 
  - entry.content_dir|trim != ''
```